src/app/shared/reactions/reactions.component.ts

```diff
@@ -19,20 +20,35 @@ export class ReactionsComponent implements OnChanges {
   @Input() showAmounts: boolean = true;
   @Input() showBar: boolean = true;
   @Input() smallIcons: boolean = false;
+  @Input() reactions: { [id: string]: number };
```

Comment:
Type can be changed to a more specific one:
```@Input() reactions: { [id: ReactionType]: number };```

This way the key can only be value of the `ReactionType` enum.

Reply:
Done.